ql/src/test/queries/clientpositive/external_jdbc_table2.q

```diff
@@ -7,43 +6,43 @@ CREATE TEMPORARY FUNCTION dboutput AS 'org.apache.hadoop.hive.contrib.genericudf
 
 FROM src
 SELECT
-dboutput ('jdbc:derby:;databaseName=${system:test.tmp.dir}/test_derby_auth1;create=true','user1','passwd1',
+dboutput ('jdbc:derby:;databaseName=${system:test.tmp.dir}/test_derby_auth1_1;create=true','user1','passwd1',
```

Comment:
Curious as to why this changed?

Reply:
I changed it to avoid a clash with other test in the temp directory, which I believe was causing HIVE-23910.